operator/cmd/mesh/operator-remove.go

```diff
@@ -78,16 +87,20 @@ func operatorRemove(cmd *cobra.Command, args *RootArgs, orArgs *operatorRemoveAr
 		l.LogAndFatal(err)
 	}
 
-	// If the user is performing purge but also specified a revision, an error message
-	// should be displayed and do nothing
+	// If the user is performing purge but also specified a revision, we should warn
+	// that the purge will still remove all resources
 	if orArgs.purge && orArgs.revision != "" {
 		orArgs.revision = ""
-		l.LogAndFatal("At most one of the --revision or --purge flags could be set\n")
-	} else if orArgs.revision == "default" {
-		orArgs.revision = ""
+		l.LogAndPrint("Purge remove will remove all Istio operator controller, ignoring the specified revision\n")
+	}
+
+	var installed bool
+	if orArgs.revision == "default" {
```

Comment:
nit: this if/else check seems to be redundant, I think we can set the revision to be `""` if it's `default` before passing `orArgs.revision` into `isControllerInstalled `, then we won't need this if/else check.
```go
if orArgs.revision == "default {
    orArgs.revision = ""
}
```

Reply:
I see. NVM I think it's fine to leave it like this. Thanks for the contribution!

Reply:
Thank you for your long and serious review